### ISSUE TITLE
nhmmscan aliscoresout segfault

### ISSUE THREAD

**Issue by bernt-matthias** [EddyRivasLab/hmmer#190]
When executing 

```
hmmpress MADE1.hmm
nhmmscan --aliscoresout aliscoresout   MADE1.hmm dna_target.fa
```

I get a segfault (hmmer 3.3). Data can be found here: https://github.com/galaxyproject/tools-iuc/tree/master/tools/hmmer3/test-data

**Comment by cryptogenomicon** [comment]
Confirmed, I can reproduce it. I'll look into it asap.

**Comment by bernt-matthias** [comment]
Another minor point: `nhmmscan` reports `# Program:         hmmscan` in the footer.

**Comment by cryptogenomicon** [comment]
I've fixed both issues in commit cdfb1baa to our develop branch. The fix will appear in our next HMMER3 release. In the meantime, the workaround is to just not use the `nhmmscan --aliscoresout` option. This option works in `nhmmer`, but was not correctly hooked up in `nhmmscan`. It's an experimental option that we've only used in testing anyway. The fix removes the option from `nhmmscan` and its documentation.

Thank you for your report!
